mon-pix/app/templates/terms-of-service.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="terms-of-service-form">
 
     <div class="terms-of-service-form__logo">
-      <a href={{urlHome}} title="Page d'accueil">
+      <a href={{homeUrl}} title="Page d'accueil">
```

Comment:
je crois qu'il y a un problème ici: on utilise `urlHome/homeUrl` mais il n'a jamais été défini côté js

Reply:
j'ai rajouté un commit `Get homeUrl for cgu page`, à fixup avec le commit `Compute home url to redirect to correct homepage` si c'est ok

Reply:
ok je fixup